ui/raidboss/common_replacement.js

```diff
@@ -4,14 +4,14 @@
 let commonReplacement = {
   '(?<=00:0839:)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?': {
     de: 'Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?$1 schließt',
-    cn: '距$1被封锁还有',
     fr: 'Fermeture d(?:e|u|es) $1 dans',
     ja: '$1の封鎖まであと',
+    cn: '距$1被封锁还有',
     ko: '15초 후에 $1(이|가) 봉쇄됩니다',
   },
   'is no longer sealed': {
     de: 'öffnet sich erneut',
-    fr: 'Ouverture ',
```

Comment:
-  the space was he voluntary

Reply:
I'm not sure that this matters either way.  This is trying to match part of the text when a zone is no longer sealed.

Reply:
I remove it as a precaution, if that does not create a problem.

Reply:
A precaution against what? Sorry, I just don't understand your intention.

Reply:
If it is not useful to delete this space, I can cancel this request. Sorry for the difficulty of comprehension.

Reply:
It was more just me trying to understand what the goal was or if you had noticed any problems.  It sounds like this is working, and I think it looks fine to me.  I will just close this pull request then, if you don't mind.